javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+
+let i = 4;
+let j = 1000;
+
+const limitedInterval = setInterval(() => {
+  $countdownDisplay.innerHTML = `${--i}`;
+
+  if (i === 0) {
+    $countdownDisplay.innerHTML = `~Earth Beeeelooowww Us~`;
+    clearInterval(limitedInterval);
+  }
+
+  if (j <= 4000) {
+    j += 1000;
+  }
```

Comment:
What is the point of this? We should just be running this function every 1 second or 1000ms. You shouldn't be changing how long this should be running